<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DockFiltres</class>
 <widget class="QWidget" name="DockFiltres">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>246</width>
    <height>278</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Filtres</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QGroupBox" name="matiere">
     <property name="title">
      <string>Matière</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout"/>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="semestre">
     <property name="title">
      <string>Semestre</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="s1">
        <property name="text">
         <string>S1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="s2">
        <property name="text">
         <string>S2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="s3">
        <property name="text">
         <string>S3</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="s4">
        <property name="text">
         <string>S4</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="controleSurveilles">
     <property name="text">
      <string>Les contrôles que je surveille</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="voirMesMatieres">
     <property name="text">
      <string>Mes matières</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="voirMatiereResponsable">
     <property name="text">
      <string>Matières ou je suis responsable</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="spacerBas">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>matiere</tabstop>
  <tabstop>semestre</tabstop>
  <tabstop>s1</tabstop>
  <tabstop>s2</tabstop>
  <tabstop>s3</tabstop>
  <tabstop>s4</tabstop>
  <tabstop>controleSurveilles</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>s1</sender>
   <signal>toggled(bool)</signal>
   <receiver>DockFiltres</receiver>
   <slot>semestreSelectionne(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>35</x>
     <y>89</y>
    </hint>
    <hint type="destinationlabel">
     <x>161</x>
     <y>86</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>s2</sender>
   <signal>toggled(bool)</signal>
   <receiver>DockFiltres</receiver>
   <slot>semestreSelectionne(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>87</x>
     <y>111</y>
    </hint>
    <hint type="destinationlabel">
     <x>83</x>
     <y>132</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>s3</sender>
   <signal>toggled(bool)</signal>
   <receiver>DockFiltres</receiver>
   <slot>semestreSelectionne(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>87</x>
     <y>137</y>
    </hint>
    <hint type="destinationlabel">
     <x>83</x>
     <y>132</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>s4</sender>
   <signal>toggled(bool)</signal>
   <receiver>DockFiltres</receiver>
   <slot>semestreSelectionne(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>87</x>
     <y>163</y>
    </hint>
    <hint type="destinationlabel">
     <x>83</x>
     <y>132</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>controleSurveilles</sender>
   <signal>toggled(bool)</signal>
   <receiver>DockFiltres</receiver>
   <slot>surveillanceSelectionne(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>83</x>
     <y>200</y>
    </hint>
    <hint type="destinationlabel">
     <x>83</x>
     <y>132</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>voirMesMatieres</sender>
   <signal>toggled(bool)</signal>
   <receiver>DockFiltres</receiver>
   <slot>voirMesMatieresSelectionne(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>91</x>
     <y>226</y>
    </hint>
    <hint type="destinationlabel">
     <x>91</x>
     <y>132</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>voirMatiereResponsable</sender>
   <signal>toggled(bool)</signal>
   <receiver>DockFiltres</receiver>
   <slot>voirMatieresResponsableSelectionne(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>147</x>
     <y>252</y>
    </hint>
    <hint type="destinationlabel">
     <x>147</x>
     <y>138</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>semestreSelectionne(bool)</slot>
  <slot>matiereSelectionne()</slot>
  <slot>surveillanceSelectionne(bool)</slot>
  <slot>voirMesMatieresSelectionne(bool)</slot>
  <slot>voirMatieresResponsableSelectionne(bool)</slot>
 </slots>
</ui>
